instrumentation/rxjava-2.0/javaagent/src/main/java/io/opentelemetry/instrumentation/rxjava2/RxJava2InstrumentationModule.java

```diff
@@ -41,15 +43,23 @@ public RxJava2InstrumentationModule() {
     @Override
     public Map<? extends ElementMatcher<? super MethodDescription>, String> transformers() {
       return Collections.singletonMap(
-          isTypeInitializer(),
-          RxJava2InstrumentationModule.class.getName() + "$RxJavaPluginsAdvice");
+          isMethod(), RxJava2InstrumentationModule.class.getName() + "$RxJavaPluginsAdvice");
     }
   }
 
   public static class RxJavaPluginsAdvice {
+    @Advice.OnMethodEnter
+    public static int trackCallDepth() {
```

Comment:
This is a terrible solution! For now just sending to demonstrate that previously the assembly wasn't getting called. The change to disable executor instrumentation for rxjava tasks caused this test to fail until I fixed this part.

Reply:
If decorating every method is really the only way, I'd at least use a `ClassValue<AtomicBoolean>` instead of calldepth.